assets/chat/js/chat.js

```diff
@@ -755,7 +762,7 @@ class Chat {
   }
 
   addMessage(message, win = null) {
-    // eslint-disable-next-line no-param-reassign
+    // eslint-disable-next-line no-param-reassign, curly
```

Comment:
You can convert this to a curly if statement as well. No need to disable.

Reply:
Added to a curly block